webpack.config.js

```diff
@@ -11,26 +11,15 @@ var commonConfig = require('./webpack.common.config');
 var license = fs.readFileSync('lib/license-header.txt', 'utf8');
 
 module.exports = _.extend({}, _.cloneDeep(commonConfig), {
-  entry: './lib/browser/index.js',
+  mode: 'production',
```

Comment:
hah, I was meaning this flag in the last review.

Reply:
Yes webpack was upgraded to version 4 in this PR